<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICCurveEditor</class>
 <widget class="QWidget" name="ICCurveEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="lineButton">
       <property name="styleSheet">
        <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="curveButton">
       <property name="styleSheet">
        <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}</string>
       </property>
       <property name="text">
        <string>Curve</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="point1Group">
       <property name="title">
        <string>Point-1</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="verticalSpacing">
         <number>14</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>X1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="y1PosLineEdit">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="zPosLineEdit">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Y1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="x1PosLineEdit">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="setInPoint1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Set In</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="point2Group">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Point-2</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>14</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>X1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="zPosLineEdit_2">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="angleLineEdit">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Y1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="x1PosLineEdit_2">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="setInPoint2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Set In</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="ICLineEditWithVirtualNumericKeypad" name="y1PosLineEdit_2">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">0.00</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Dir</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QComboBox" name="dirBox">
          <item>
           <property name="text">
            <string>PP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RP</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICLineEditWithVirtualNumericKeypad" name="speedEdit">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Delay:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ICLineEditWithVirtualNumericKeypad" name="delayEdit">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">0.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
